### DISCUSSION TITLE
An article in russian about extending MathPHP

### DISCUSSION THREAD

**Discussion by Beakerboy** [markrogoyski/math-php#446] [General]
https://top-technologies.ru/article/view?id=38404

translation:
https://top--technologies-ru.translate.goog/article/view?id=38404&_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en-US

**Comment by markrogoyski** [comment]
Interesting. It looks like maybe an academic journey of some sort. Thanks for sharing!

**Reply by Beakerboy** [reply]
IMO it doesn’t look too impressive, but at least people are referencing the project.
